### PR TITLE
BRINGS THE GREAT DEVTEST

### PR DESCRIPTION
**Pull request by Laboredih123**
[MAPPING][QOL]

## About the PR 
Adds a new devmap for developers to test features and various other things.
Includes: Side areas to test pods and subs, a nuclear reactor, a thermoelectric engine, a singularity, material science, mechanics, mining, ghostdrone creator, working cargo, tiny AI chamber, botany, ranch, angler equipment, kitchen, bar, chemistry, telesci, artsci, termOS, teleporter, medical with cryo and surgery, robotics, cryogenic freezer, toxins, a large test chamber with essentially bomb proof walls, and perhaps more in the future.

## Why's this needed? 
Atlas has grown too large and has lessened its usefulness for development. A new map is needed to let developers test anything they wish easily, which is this one hopefully.

### CODE REVIEW

**Comment by ZeWaka** [comment]
need pictures and better description

**Comment by Laboredih123** [comment]
> need pictures and better description

Image added. Unsure of how to better description, tried anyway.

**Comment by TheTyrant25** [comment]
presumably mention all features that are in the devtest map so that people viewing this pr in the future can see what was originally added vs what got changed (if any) later down the line

**Review by TheTyrant25 — CHANGES_REQUESTED** [review]
image
you seem to have duplicated the file into unused and the main map folder btw

**Comment by Laboredih123** [comment]
shite

**Review by TheTyrant25 — APPROVED** [review]

**Comment by Laboredih123** [comment]
> presumably mention all features that are in the devtest map so that people viewing this pr in the future can see what was originally added vs what got changed (if any) later down the line

ok i listed all the things

**Comment by Laboredih123** [comment]
!merge upstream

**Comment by Laboredih123** [comment]
is that meant to do something

**Comment by Laboredih123** [comment]
oh i stupif

**Comment by Laboredih123** [comment]
!merge_upstream

**Review by Tarmunora — CHANGES_REQUESTED** [review]
Security area can be dropped - there's really nothing I can see needing to test wrt sec stuff that isn't really trivially spawned anyways - at most, the sec vendor an a table of requisition tokens
Clown hole, gnome chompsky, station pets. and other jokes/eastereggs/"fun things" can be dropped - kinda just bloat

**Comment by Laboredih123** [comment]
time to kill stuff

**Comment by Laboredih123** [comment]
looks like i forgot to update the image

**Review by Tarmunora — APPROVED** [review]

File: _std/__build.dm [hunk 1]
```diff
@@ -127,6 +127,7 @@ o+`        `-` ``..-:yooos-..----------..`
 //#define MAP_OVERRIDE_CLARION			// Used to be Destiny's Alt. By Dionsu and a69andahalf.
 //#define MAP_OVERRIDE_OSHAN				// Oshan by committee
 //#define MAP_OVERRIDE_NADIR				// Nadir Extraction Site by Kubius
+//#define MAP_OVERRIDE_DEVTEST			// Developer Testing map by cringe
```
[the remark this answers is not in the document]
Comment (Laboredih123) [reply _std/__build.dm:130]: ight then i suppose

**Review by Azrun — APPROVED** [review]

**Pull request merged**